DoctorRanker.py

```diff
@@ -1,31 +0,0 @@
-import pandas as pd
```

Comment:
is there a way to run this code? you might wanna test that. if you install python locally, you could set up a [virtualenv](https://virtualenv.pypa.io/en/latest/), install pandas/numpy, and then try running it with `python3 DoctorRanker.py` or something; the people will surely try to test your code at least. that can catch some random syntax errors or things that i can't find right away since i believe that the code largely works fine since you tested in colab.